### DISCUSSION TITLE
Add ipython magic

### DISCUSSION THREAD

**Discussion by lorentzenchr** [wjakob/nanobind#333] [Q&A] [answered]
Cython has a neat `%%cython` ipython magic command, see https://cython.readthedocs.io/en/latest/src/quickstart/build.html#using-the-jupyter-notebook. I find that very useful to lower the entry barrier and fast playing around.
Is it conceivable to add something similar to nanobind?

**Comment by wjakob** [comment, marked as answer]
Let's keep the issue tracker for actual issues (this sounds to me like more of a feature request). Regarding your question: certainly it's conceivable, but somebody has to sit down and actually implement it ;). It's not a priority for me, and I daresay it's perhaps even a bit too specific to be part of this repository.

See also: https://github.com/aldanor/ipybind

**Reply by lorentzenchr** [reply]
Thanks for the fast answer.
